spec/system/admin/order_cycles/complex_creating_specific_time_spec.rb

```diff
@@ -67,6 +67,9 @@
     click_button 'Create'
     expect(page).to have_content 'Your order cycle has been created.'
 
+    # wait for the page to load
+    expect(page).to have_no_content "Loading..."
+
```

Comment:
Hey @jibees I only see a change in the spec - I guess there might be other commits in the PR?

Reply:
> (Am I sure to be sure? Not sure...) 

:-D

> I'm wondering what kind of change you were expecting?

I might have misinterpreted the issue, or maybe we have more than one. But my understanding was that this is not related to the spec, so I would expect a change in the code itself - but I might be wrong.

In any case, I've tested this branch locally with our script and got `74 of 100 passed (74%)`. The spec failures were the same as before:

```
     Failure/Error: expect(page).to have_field "order_cycle_outgoing_exchange_0_pickup_time"
       expected to find field "order_cycle_outgoing_exchange_0_pickup_time" that is not disabled but there were no matches
```

We still get this error on the console `Cannot read properties of undefined` - it's a pain to catch, but when it appears, the dropdown does not appear, and the spec fails:

![image](https://user-images.githubusercontent.com/49817236/208973556-397f3838-c6b2-4cac-bd91-b672e123521c.png)

I think this is appearing within the helpers:
```
add_supplier_with_fees
add_distributor_with_fees
```